.terraform-docs.yml

```
@@ -0,0 +1,48 @@
+formatter: "" # this is required
```

J'ai une préférence pour ne pas mettre de " dans les yaml (sauf si c'est requis) 

Peux-tu mettre le fichier dans ce format 

```suggestion
formatter:  # this is required
```